Assets/Scripts/Interactable/Upgradable.cs

```diff
@@ -23,6 +23,7 @@ public class Upgradable : Interactable
     private List<GameObject> _chipInstances = new List<GameObject>();
     private InteractableHealth _health;
     protected Upgrade _selectedUpgrade;
+    protected int _chipLevel;
```

Comment:
Aren't `Upgradable` always supposed to have 2 chips?

Reply:
Both chips must be of the same level.